modules/__tests__/enhancer-test.js

```diff
@@ -38,6 +38,19 @@ describe('Enhancer', () => {
     );
   });
 
+  it('receives the given props', () => {
+    class Composed extends Component {
+      constructor (props) {
+        super(props);
+      }
+    }
+    var Enhanced = Enhancer(Composed);
+
+    var instance = new Enhanced({foo: 'bar'});
```

Comment:
Do you mean something like this
```var element = <Enhanced/>;
React.renderToString(element);```
instead of this
```var instance = new Enhanced();```?